src/store/actions.ts

```diff
@@ -31,7 +31,7 @@ function loading(type: 'dataset' | 'uniqueValues') {
 export function formatError(error: any): BackendError {
   return typeof error === 'string'
     ? { type: 'error', message: error.toString() }
-    : { type: 'error', ...error };
+    : { type: 'error', ...error, message: error.message ?? error.errmsg };
```

Comment:
Hm... I think that should be harmonized back-end side, or we will have to implement a different rule for every possible backend. Let's discuss what we choose to implement.

Reply:
Totally agree!